.github/workflows/devel.yaml

```diff
@@ -13,15 +13,29 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Build Image
-        run: |
-          IMG=awx-operator:devel make docker-build
+      - name: Log into registry ghcr.io
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d    # v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
 
-      - name: Push To Quay
-        uses: redhat-actions/push-to-registry@v2.1.1
+      - name: Log into registry quay.io
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d    # v3.0.0
         with:
-          image: awx-operator
-          tags: devel
           registry: quay.io/ansible/
           username: ${{ secrets.QUAY_USER }}
           password: ${{ secrets.QUAY_TOKEN }}
+
+
+      - name: Build and Store Image @ghcr
+        run: |
+          IMG=ghcr.io/${{ github.repository_owner }}/${{ github.repository }}:${{ github.sha }} make docker-buildx
+
+
+      - name: Publish Image to quay.io/ansible/awx-operator:devel
+        run: |
+          docker buildx imagetools create \
+            ghcr.io/${{ github.repository_owner }}/${{ github.repository }}:${{ github.sha }} \
+            --tag quay.io/ansible/${{ github.repository }}:devel
```

Comment:
Don't know if there is an issue with the linter but it inconsistent.

- 466b0cf5c784894e3a0139e5c9014401c52452f5 too many blank lines `0>1` when there were two blank lines https://github.com/ansible/awx-operator/actions/runs/7457722546
- 4d7936515c89ab9c0be6fb985fd968640874bf26 no blank line at end of file `no new line character at the end of file` https://github.com/ansible/awx-operator/actions/runs/7470471612